### PR TITLE
Feature/catd 89 campaign automation service   sd

### PR DESCRIPTION
**Pull request by ndy40**
Added the following missing API Endpoints

- Screens, ScreenVariation, Widgets_by_audience and `screens/{id}/widgets`
- Widgets, WidgetVariation

### CODE REVIEW

File: livestyled/tests/fixtures/mock_responses/ls_api/canvas/widgets_by_audience.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                ]
+            }
+        }
+    ],
+    "hydra:totalItems": 1,
+    "hydra:view": {
+        "@id": "/v4/canvas/screens/413/widgets_by_audience?audience=37&timestamp=1643907912.0897183",
+        "@type": "hydra:PartialCollectionView"
+    }
+}
```
Comment (tomrutherford-livestyled) [inline livestyled/tests/fixtures/mock_responses/ls_api/canvas/widgets_by_audience.json:32]: Can you add new lines for these files please.

File: livestyled/models/screen.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+
+class ScreenTranslation:
+    def __init__(self, language: str, title: str, **kwargs):
+        self.language = language
+        self.title = title
+
+        if kwargs:
+            for k, v in kwargs.items():
+                setattr(self, k, v)
```
[the remark this answers is not in the document]
Comment (ndy40) [reply livestyled/models/screen.py:12]: Yes it is for setting dynamic properties. The only required fields are language and title.

[the remark this answers is not in the document]
Comment (tomrutherford-livestyled) [reply livestyled/models/screen.py:12]: Ok thanks, how comes this is the only model that needs this?

[the remark this answers is not in the document]
Comment (ndy40) [reply livestyled/models/screen.py:12]: The Screen model on v3 uses a JSON column. Some of our Translation models are tables on their own.

**Review by tomrutherford-livestyled — APPROVED** [review]

**Pull request merged**